driver-core/src/main/com/mongodb/client/model/expressions/StringExpression.java

```diff
@@ -21,4 +21,17 @@
  */
 public interface StringExpression extends Expression {
 
+    StringExpression toLower();
+
+    StringExpression toUpper();
+
+    Expression concat(StringExpression concat);
```

Comment:
Quick question: why don't any of these return a narrower Expression type?

Reply:
Autogen error.